client/src/App.css

```diff
@@ -1,4 +1,12 @@
-#root {
+body {
+  font-family: Lato, sans-serif;
+}
+
+h1 {
+  font-family: Arvo, serif;
+}
+
+/* #root {
```

Comment:
Eventually, once you are sure you don't need these, you will want to delete these unused rules.